Source/WebCore/rendering/RenderListMarker.cpp

```diff
@@ -2025,9 +2002,15 @@ StringView RenderListMarker::textWithoutSuffix() const
     return StringView { m_textWithSuffix }.left(m_textWithoutSuffixLength);
 }
 
-CSSCounterStyle* RenderListMarker::counterStyle() const
+RefPtr<CSSCounterStyle> RenderListMarker::counterStyle() const
+{
+    return document().counterStyleRegistry().resolvedCounterStyle(style().listStyleType().identifier);
+}
+
+bool RenderListMarker::markerNeedsWidthAdjust() const
```

Comment:
```suggestion
bool RenderListMarker::widthUsesMetricsOfPrimaryFont() const
```

What do you think of this name?

Reply:
what do you mean with the `MetricsOfPrimaryFont` bit?

Reply:
We seem to be using `font.metricsOfPrimaryFont()` for these cases.